components/blocks/VerticalCards.template.ts

```diff
@@ -0,0 +1,33 @@
+import type { TinaTemplate } from 'tinacms';
+
+export const verticalCardsTemplate: TinaTemplate = {
+  label: 'Vertical Cards',
+  name: 'verticalCards',
+  ui: {
+    previewSrc: '',
+  },
+  fields: [
+    {name: 'title', label: 'Title', type: 'string'},
+    {
+      name: 'cardItems',
+      label: 'Card Items',
+      type: 'object',
+      list: true,
+      ui: {
+        itemProps: (item) => ({
+          key: item.id,
+          label: item.headline,
+        }),
+      },
+      fields: [
+        { name: 'headline', label: 'Headline', type: 'string' },
+        { name: 'description', label: 'Description', type: 'string' },
+        { name: 'timeDate', label: 'Time and Date', type: 'string' },
```

Comment:
Shouldn't this field be a `datetime` rather than a `string`?

Reply:
no because dateTime only supports a singular date, we want to be able to have the option to have from x -> y dates or just x date